lib/kanban/process.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Kanban.Process do
+  @moduledoc """
+  Process to be run
+  """
+
+  use GenServer
+
+  @doc false
+  def start_link(state) do
+    GenServer.start_link(__MODULE__, state, name: __MODULE__)
+  end
+
+  @impl true
+  def init(state) do
+    {:ok, state}
+  end
+
+  @impl GenServer
+  def handle_call({:+, value}, _from, state) do
```

Comment:
С гардом сообщение об ошибке будет ближе к месту ошибки и втрое более внятное.

```suggestion
  def handle_call({:+, value}, _from, state) when is_number(value) do
```